ci/default.nix

```
@@ -75,6 +75,23 @@ let
           includes = [ "*" ];
           priority = 1;
         };
+
+        # TODO: Upstream this into treefmt-nix eventually:
+        #   https://github.com/numtide/treefmt-nix/issues/387
+        settings.formatter.markdown-code-runner = {
+          command = pkgs.lib.getExe pkgs.markdown-code-runner;
+          options = [
+            "--config=${
+              pkgs.writers.writeTOML "markdown-code-runner-config" {
+                presets.nixfmt = {
+                  language = "nix";
+                  command = [ (pkgs.lib.getExe pkgs.nixfmt) ];
+                };
+              }
+            }"
+          ];
+          includes = [ "*.md" ];
+        };
```

I have an irrational preference to use `let ... in` blocks instead of mixing languages in an interpolated string. Like this:

```suggestion
        settings.formatter.markdown-code-runner = {
          command = pkgs.lib.getExe pkgs.markdown-code-runner;
          options =
            let
              mcrToml = pkgs.writers.writeTOML "markdown-code-runner-config" {
                presets.nixfmt = {
                  language = "nix";
                  command = [ (pkgs.lib.getExe pkgs.nixfmt) ];
                };
              };
            in
            [ "--config=${mcrToml}" ];
          includes = [ "*.md" ];
        };
```

This is particularly ironic on a PR about formatting languages inside of string interpolations.